packages/core/src/browser/style/sidepanel.css

```diff
@@ -86,7 +86,7 @@
 }
 
 .p-TabBar.theia-app-left .p-TabBar-tab.p-mod-current.theia-mod-active {
-    border-left: var(--theia-panel-border-width) solid var(--theia-focusBorder);
+    border-left: var(--theia-panel-border-width) solid var(--theia-selection-background)
```

Comment:
we should be careful with border colors

general guideline is:
- light and dark themes should not color borders, but user background colors for constrast
- hc theme should use the same background color, but use focusBorder stlye for contrast

We should check that such changes don't break it.

Reply:
Thank you for your response. I tested the new class with all the color themes and it works fine with all of them and as far as I can test, it doesn't break anything. 

it would be great if you could verify it @akosyakov